weaviate/str_enum.py

```diff
@@ -0,0 +1,19 @@
+# MetaEnum and BaseEnum are required to support `in` statements:
```

Comment:
Was there a circular dependency requiring this move out of `util.py` and into `str_enum.py`?

Reply:
yes, exactly